### PR TITLE
Jp/264 split up repo

### PR DESCRIPTION
**Pull request by jonpither**
PR ready to merge. 

Initial work on splitting up fixtures, as it glues the various would-be-modules together. I got sidetracked and decided to tackle #251 once and for all.

### CODE REVIEW

**Comment by jonpither** [comment]
have needed a couple of thread/sleeps, but I think this PR unearthed an existing issue that will need looking at.

**Comment by jonpither** [comment]
This refactor is also causing tx-test to intermittently fail with the below:
```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f8e19c5c7ca, pid=22883, tid=7992
#
# JRE version: OpenJDK Runtime Environment (10.0.2+13) (build 10.0.2+13)
# Java VM: OpenJDK 64-Bit Server VM (10.0.2+13, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
# Problematic frame:
# C  [liblwjgl_lmdb.so+0xe7ca]
#
# Core dump will be written. Default location: Core dumps may be processed with "/usr/lib/systemd/systemd-coredump %P %u %g %s %t %c %h %e" (or dumping to /home/jon/dev/juxt/crux/core.22883)
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

---------------  S U M M A R Y ------------

Command Line: -Dfile.encoding=UTF-8 -Dclojure.compile.path=/home/jon/dev/juxt/crux/target/classes -Dcrux.version=19.06-1.1.1-alpha-SNAPSHOT -Dclojure.debug=false -Djava.library.path=/home/jon/dev/juxt/crux/target/native:/home/jon/dev/juxt/crux/target/native:/home/jon/dev/juxt/crux/target/native:/home/jon/dev/juxt/crux/target/native clojure.main -i /tmp/form-init12080172631445436664.clj

Host: Intel(R) Core(TM) i7-6560U CPU @ 2.20GHz, 4 cores, 15G, Arch Linux
Time: Mon Jun 24 08:29:23 2019 BST elapsed time: 232 seconds (0d 0h 3m 52s)

---------------  T H R E A D  ---------------

Current thread (0x00007f8eaef35000):  JavaThread "crux.tx.event-log-consumer-thread" [_thread_in_native, id=7992, stack(0x00007f8e195d7000,0x00007f8e196d8000)]

Stack: [0x00007f8e195d7000,0x00007f8e196d8000],  sp=0x00007f8e196d6210,  free space=1020k
Native frames: (J=compiled Java code, A=aot compiled Java code, j=interpreted, Vv=VM code, C=native code)
C  [liblwjgl_lmdb.so+0xe7ca]

Java frames: (J=compiled Java code, j=interpreted, Vv=VM code)
J 14177  org.lwjgl.util.lmdb.LMDB.nmdb_cursor_get(JJJI)I (0 bytes) @ 0x00007f8e9b83a51f [0x00007f8e9b83a460+0x00000000000000bf]
J 14622 c2 crux.kv.lmdb.LMDBKvIterator.seek(Ljava/lang/Object;)Ljava/lang/Object; (118 bytes) @ 0x00007f8e9b896bf8 [0x00007f8e9b896a60+0x0000000000000198]
J 13554 c2 crux.lru.CachedIterator.seek(Ljava/lang/Object;)Ljava/lang/Object; (126 bytes) @ 0x00007f8e9b7a658c [0x00007f8e9b7a63c0+0x00000000000001cc]
J 17398 c1 crux.moberg$seek_message.invokeStatic(Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object; (280 bytes) @ 0x00007f8e95b639ec [0x00007f8e95b63180+0x000000000000086c]
J 17397 c1 crux.moberg$seek_message.invoke(Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object; (13 bytes) @ 0x00007f8e95b62c34 [0x00007f8e95b62bc0+0x0000000000000074]
J 17860 c1 crux.tx$event_log_consumer_main_loop$fn__10887.invoke()Ljava/lang/Object; (1113 bytes) @ 0x00007f8e94332604 [0x00007f8e94331640+0x0000000000000fc4]
j  crux.tx$event_log_consumer_main_loop.invokeStatic(Ljava/lang/Object;)Ljava/lang/Object;+344
j  crux.tx$event_log_consumer_main_loop.invoke(Ljava/lang/Object;)Ljava/lang/Object;+3
j  crux.tx$start_event_log_consumer$fn__10901.invoke()Ljava/lang/Object;+56
j  clojure.lang.AFn.run()V+1
J 16449 c1 java.lang.Thread.run()V java.base@10.0.2 (17 bytes) @ 0x00007f8e93fa621c [0x00007f8e93fa60e0+0x000000000000013c]
v  ~StubRoutines::call_stub
```

File: docs/query_test_examples.clj [hunk 1]
```diff
@@ -748,132 +748,7 @@
   [:p4 :NY #inst "2019-01-02" :na]}
 ;; end::bitempr[]
 
-(t/deftest test-bitemp-query-from-indexing-temporal-data-using-existing-b+-trees-paper
```
Comment (hraberg) [inline docs/query_test_examples.clj:751]: Where did this test go? Is it simply just removed?

**Review by hraberg — APPROVED** [review]
We know there's a flaky test here, and that an example test used by the docs have been removed, but we'll fix both on master.

**Pull request merged**